### ISSUE TITLE
Add functionality to "Add User" && "Remove User" button

### ISSUE THREAD

**Issue by d22mikwe** [HGustavs/LenaSYS#15562]
After #15491 functionality should be added.

Intended functionality is to be able to search for all users in db table user in the dropdown menu.
Then when a user i chosen and save button is pressed this user should be added to that course in db table user_course.
Also intended functionality is to remove users using the same dropdown list.
May split this into 2 issues

Add functionality for this.
